mmdet3d/models/middle_encoders/sparse_encoder.py

```diff
@@ -30,8 +31,10 @@ def __init__(self,
                  encoder_channels=((16, ), (32, 32, 32), (64, 64, 64), (64, 64,
                                                                         64)),
                  encoder_paddings=((1, ), (1, 1, 1), (1, 1, 1), ((0, 1, 1), 1,
-                                                                 1))):
+                                                                 1)),
+                 block_type='submblock'):
```

Comment:
I suggest to use the term `conv_module` rather than `submblock` to distinguish it from a block since it is just a conv-bn-relu module.